llvm/test/tools/llvm-objcopy/ELF/Inputs/srec-elf-sections.yaml

```diff
@@ -0,0 +1,48 @@
+# FIXME: Currently the sections in this file are ignored and we
```

Comment:
Tests from the original ihex patch (https://reviews.llvm.org/D60270) may be useful.

Reply:
The tests in this patch are based on the ones in the ihex patch